packages/react/src/hooks/contracts/useContractRead.ts

```diff
@@ -127,7 +127,7 @@ export function useContractRead(
     watch,
   ])
 
-  return useQuery(queryKey_, queryFn, {
+  const contractReadQuery = useQuery(queryKey_, queryFn, {
```

Comment:
Would react-query's `select` config option work instead of capturing the whole result? That way, the return value still takes advantage of the tracked behavior.

```
select: (data: TData) => unknown
Optional
This option can be used to transform or select a part of the data returned by the query function.
```

Reply:
Forwarding from convo: I tried using `select`, but it doesn't work b/c of the data comparison bug i found in RQ lol

Reply:
Lol figures. Maybe worthwhile opening up something over there and we try to find another workaround. Losing the tracked result isn't ideal to fix this case. What do you think?

Reply:
yeah sounds good. 

https://github.com/tannerlinsley/react-query/pull/3669